icons/folder_type_windows.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>folder_type_windows</title><path d="M27.9,6H18.1l-2,4H5V27H30V6Zm.1,4H20.2l1-2H28Z" style="fill:#55a1bf"/><path d="M10,12.974l8.582-1.166v8.253L10,20.109Zm8.577,8.037.008,8.261L10.006,28.1v-7.14Zm1.041-9.355L31,10v9.956l-11.379.089ZM31,21.089V31L19.618,29.4,19.6,21.07Z" style="fill:#00adef"/></svg>
```

Comment:
`Windows 10` blue color is a [little bit darker](https://commons.wikimedia.org/wiki/File:Windows_10_Logo.svg). We're using the `Windows 8` blue here.

Reply:
Adjusting....